Bottom: 7b6fb2c79abc1a0050f19a2aae0d0e1a9cc19220
Top:    bbf2ae45fd007a5c6ac80a98de28ecd200156a21
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 01:51:07 +0000

Refresh of mymem-c-fix-incorrect-types-pa

---

diff --git a/src/mymem.c b/src/mymem.c
index f7939e7..c79e068 100644
--- a/src/mymem.c
+++ b/src/mymem.c
@@ -475,10 +475,10 @@ GetDrillElementMemory (DrillTypePtr Drill)
     {
       Drill->ElementMax += STEP_ELEMENT;
       element = (ElementTypePtr *)realloc (element,
-                         Drill->ElementMax * sizeof (ElementTypeHandle));
+                         Drill->ElementMax * sizeof (ElementTypePtr));
       Drill->Element = element;
       memset (element + Drill->ElementN, 0,
-	      STEP_ELEMENT * sizeof (ElementTypeHandle));
+	      STEP_ELEMENT * sizeof (ElementTypePtr));
     }
   return (element + Drill->ElementN++);
 }
@@ -497,9 +497,9 @@ GetDrillPinMemory (DrillTypePtr Drill)
   if (Drill->PinN >= Drill->PinMax)
     {
       Drill->PinMax += STEP_POINT;
-      pin = (PinTypePtr *)realloc (pin, Drill->PinMax * sizeof (PinTypeHandle));
+      pin = (PinTypePtr *)realloc (pin, Drill->PinMax * sizeof (PinTypePtr));
       Drill->Pin = pin;
-      memset (pin + Drill->PinN, 0, STEP_POINT * sizeof (PinTypeHandle));
+      memset (pin + Drill->PinN, 0, STEP_POINT * sizeof (PinTypePtr));
     }
   return (pin + Drill->PinN++);
 }
